app/views/layouts/application.html.erb

```diff
@@ -86,9 +86,9 @@
         <span class="icon-bar"></span>
         <span class="icon-bar"></span>
       </button>
-      <a class="navbar-brand" href="/rooms">
+      <a class="navbar-brand" href="/">
         <% if company_logo == nil %>
-          <a class="navbar-brand" href="/rooms">CRS2000</a>
+          <a class="navbar-brand" href="/">CRS2000</a>
```

Comment:
Could we change this to be a `link_to` tag that goes to the `root_path` instead?